Sources/PrimerSDK/Classes/PCI/Tokenization View Models/FormsTokenizationViewModel/CardFormPaymentMethodTokenizationViewModel.swift

```diff
@@ -328,7 +328,58 @@ class CardFormPaymentMethodTokenizationViewModel: PaymentMethodTokenizationViewM
             })
         }
 
-        super.start()
```

Comment:
On the face of it I dont like introducing this repetition for the sake of one line difference. Understand a more elegant solution might not be simple, but were there any other approaches considered?

Would there be any way we could take `self.uiManager.primerRootViewController?.enableUserInteraction(true)` out of the `super` implementation, and inject it/execute it from the contexts it is needed.

Also begs the question: is `self.uiManager.primerRootViewController?.enableUserInteraction(true)` actually required in the `super` method? Will other flows break without this? @BorisNikolic

Reply:
Ticketed [here](https://primerapi.atlassian.net/browse/ACC-4433)